pkgs/development/python-modules/hledger-utils/default.nix

```diff
@@ -0,0 +1,48 @@
+{lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, setuptools-scm
+, rich
+, pandas
+, scipy
+, psutil
+, matplotlib
+, drawilleplot
+, asteval
+}:
+
+buildPythonPackage rec {
+  pname = "hledger-utils";
+  version = "1.12.1";
+
+  src = fetchPypi {
+    # pname = "hledger_utils";
```

Comment:
Looks like this is unneeded, the files use dashes.

https://pypi.org/project/hledger-utils/#files